src/easylink/pipeline.py

```diff
@@ -4,8 +4,8 @@
 ========
 
 This module is responsible for the ``Pipeline`` class, whose primary purpose
-is to perform validations as well as generate the Snakefile to be used by Snakemake 
-to execute the pipeline.
+is to perform validations as well as generate the Snakefile to be used by 
+`Snakemake <https://snakemake.readthedocs.io/en/stable/>`_ to execute the pipeline.
```

Comment:
For some reason I can't just do :mod:`snakemake` like I can with other external links (even when adding the url for snakemake to the intersphinx_mapping and confirming the link actually works). But this works 🤷